cursor.c

```diff
@@ -1,18 +1,16 @@
-// Copyright 2021 Google LLC
-// Copyright 2021 @filterpaper
-// Copyright 2023 Pablo Martinez (@elpekenin) <elpekenin@elpekenin.dev>
 // Copyright 2024 Guillaume Stordeur <guillaume.stordeur@gmail.com>
 // Copyright 2024 Matt Skalecki <ikcelaks@gmail.com>
 // Copyright 2024 QKekos <q.kekos.q@gmail.com>
 // SPDX-License-Identifier: Apache-2.0
```

Comment:
I think we have a license mismatch. The files all specify Apache-2.0, but the repo LICENSE file is GPL2, I think.

We should choose a license and get that all squared away.